RULES.md

```diff
@@ -1,35 +1,20 @@
 # Rules
 
-This table documents all implemented `wdl` rules (validation failures and lint
-warnings) implemented on the `main` branch of the `stjude-rust-labs/wdl`
-repository. Note that the information may be out of sync with released packages.
+This table documents all implemented `wdl` lint rules implemented on the `main` 
+branch of the `stjude-rust-labs/wdl` repository. Note that the information may 
+be out of sync with released packages.
```

Comment:
> Note that the information may be out of sync with released packages.

This is a good disclaimer to have till we start building docs from the code. That being said, we should make sure the instructions in `.github/pull_request_template.md` are up to speed and modifying the below table is part of those instructions.